guides/common/assembly_synchronizing-content-between-servers.adoc

```diff
@@ -50,4 +50,6 @@ include::modules/proc_importing-a-repository.adoc[leveloffset=+1]
 
 include::modules/proc_importing-a-repository-from-a-web-server.adoc[leveloffset=+1]
 
+include::modules/proc_enabling-custom-repositories-on-content-hosts.adoc[leveloffset=+1]
```

Comment:
This isn't the right place for this. This has nothing to do with ISS.

Reply:
It should probably go somewhere in here: https://docs.theforeman.org/nightly/Managing_Hosts/index-katello.html#Administering_Hosts_managing-hosts

Reply:
Moved to after the Cloning Hosts section.

Reply:
How about after "Changing a module stream?" Then it'll be next to other sections about host content.

Reply:
Done!